### PR TITLE
Improve the text

### PR DESCRIPTION
**Pull request by zoechi**
I'm sure there is still room for improvement.
I found it sometimes just too verbose.
Please let me know if I butchered the original intent, so we can further improve it.

### CODE REVIEW

File: docs/project_practice/how_to_build_a_todo_application_with_ribir.md [hunk 1]
```diff
@@ -4,17 +4,17 @@ sidebar_position: 1
...
 
-Now let's define the simplest Todo application data structure. I call this structure `TodoMVP`. And as a simplest Todo application has a list of what it use to save the collection of todo task at least. In Rust, we can use `Vec` structure to describe a list. And then, the list must have many task. A single task needs two fields, `done` and `description`. The `done` field use type `bool` to describe task whether or not to complete, and the `description` field use type `String` to describe task specific information.
+Now let's define the simplest Todo application data structure. I call this structure `TodoMVP`. As a simplest Todo application it has a list of what it uses to save the collection of todo tasks at least. In Rust, we can use the `Vec` structure to describe a list, and then, the list must be able hold many task. A single task needs two fields, `done` and `description`. The `done` field is of type `bool` to describe whether or not the task is completed and the `description` field is of type `String` to describe task specific information.
```
Comment (macyhan) [inline docs/project_practice/how_to_build_a_todo_application_with_ribir.md:17]: the list must be able hold many task --> the list must be able to hold many task

File: docs/project_practice/how_to_build_a_todo_application_with_ribir.md [hunk 2]
```diff
@@ -38,13 +38,13 @@ fn main() {
...
+We have a base data structure. Now we use Ribir to build the user interface part of the Todo app. The basic Todo application needs to have two features, input and display. In the input part we can use an `Input` widget to enter a description for a new task. For the display part we can use a `List` widget to render the task list. We start with adding `Input` at the top and the `List` below the input. To accomplish that we use a `Column` widget, which is a basic vertical arrangement layout widget.
 
-We call Todo to interface it `todo_widget`.
+**We call Todo to interface it `todo_widget`.**(I don't understand this part)
```
Comment (macyhan) [inline docs/project_practice/how_to_build_a_todo_application_with_ribir.md:47]: It should be:

We call the user interface of Todo `todo_widget`.

File: docs/project_practice/how_to_build_a_todo_application_with_ribir.md [hunk 3]
```diff
@@ -404,7 +404,7 @@ impl TodoMVP {
 
 ## Adding transition animation
 
-The Ribir is a moden GUI library. Let's add some animation to show. We add a task moving show when it is mounted.
+Ribir is a moden GUI library. Let's add some animation to demonstrate that. We add a task show animation when it is mounted.
```
Comment (macyhan) [inline docs/project_practice/how_to_build_a_todo_application_with_ribir.md:407]: moden--> modern 
some animation --> some animations

**Comment by zoechi** [comment]
I implemented the suggestions. Please have another look.

**Review by macyhan — APPROVED** [review]

**Review by M-Adoo — APPROVED** [review]

**Pull request merged**